sdk/servicebus/service-bus/samples/typescript/src/usingAadAuth.ts

```diff
@@ -19,34 +19,33 @@
 */
 
 import { ServiceBusClient } from "@azure/service-bus";
-import { loginWithServicePrincipalSecret } from "@azure/ms-rest-nodeauth";
+import { DefaultAzureCredential } from "@azure/identity";
 
 // Load the .env file if it exists
 import * as dotenv from "dotenv";
 dotenv.config();
 
 // Define Service Bus Endpoint here and related entity names here
 const serviceBusEndpoint =
-  process.env.SERVICE_BUS_ENDPOINT || "<your-servicebus-namespace>.servicebus.windows.net";
+  process.env.SERVICE_BUS_ENDPOINT ||
+  "<your-servicebus-namespace>.servicebus.windows.net";
```

Comment:
`sb://` ?

Reply:
It's not required in this case. The sb:// does show up in a connection string - this is just intended to be the hostname.

(is there a spot where it is required aside from the connection string?)